Gemfile

```diff
@@ -106,8 +106,8 @@ group :development do
 end
 
 group :test do
+  gem 'webmock', '~> 3.6'
```

Comment:
I'm assuming moving webmock on top didn't actually do anything?

Reply:
Indeed it did not. I was exploring possible differences between web and webtest.